op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -113,23 +225,11 @@ func Test_Operator_Fee_Constistency(gt *testing.T) {
 		env.RunFaultProofProgram(t, l2SafeHead.Number, testCfg.CheckResult, testCfg.InputParams...)
 	}
 
-	matrix := helpers.NewMatrix[any]()
+	matrix := helpers.NewMatrix[testCase]()
 	defer matrix.Run(gt)
 
-	matrix.AddTestCase(
-		"HonestClaim-OperatorFeeConstistency",
-		nil,
-		helpers.NewForkMatrix(helpers.Isthmus),
-		runIsthmusDerivationTest,
-		helpers.ExpectNoError(),
-	)
-
-	matrix.AddTestCase(
-		"JunkClaim-OperatorFeeConstistency",
-		nil,
-		helpers.NewForkMatrix(helpers.Isthmus),
-		runIsthmusDerivationTest,
-		helpers.ExpectError(claim.ErrClaimNotValid),
-		helpers.WithL2Claim(common.HexToHash("0xdeadbeef")),
-	)
+	matrix.AddDefaultTestCasesWithName("OperatorFeeConstistency-NormalTx", NormalTx, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
+	matrix.AddDefaultTestCasesWithName("OperatorFeeConstistency-DepositTx", DepositTx, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
+	matrix.AddDefaultTestCasesWithName("OperatorFeeConstistency-StateRefund", StateRefund, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
+	matrix.AddDefaultTestCasesWithName("OperatorFeeConstistency-IsthmusTransitionBlock", IsthmusTransitionBlock, helpers.NewForkMatrix(helpers.Holocene), runIsthmusDerivationTest)
```

Comment:
```suggestion
	matrix.AddDefaultTestCasesWithName("NormalTx", NormalTx, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
	matrix.AddDefaultTestCasesWithName("DepositTx", DepositTx, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
	matrix.AddDefaultTestCasesWithName("StateRefund", StateRefund, helpers.NewForkMatrix(helpers.Isthmus), runIsthmusDerivationTest)
	matrix.AddDefaultTestCasesWithName("IsthmusTransitionBlock", IsthmusTransitionBlock, helpers.NewForkMatrix(helpers.Holocene), runIsthmusDerivationTest)
```
the test name itself would already be clear